tests/jballerina-unit-test/src/test/resources/test-src/expressions/mappingconstructor/spread_op_field.bal

```diff
@@ -150,15 +150,84 @@ type Grades record {|
     int...;
 |};
 
-type Address record {
+type Address record {|
```

Comment:
I would leave this as is and add a new record since this was already used in a different test.